learn/inner_workings/datatypes.mdx

```diff
@@ -12,14 +12,18 @@ String tokenization is the process of **splitting a string into a list of indivi
 
 A string is passed to a tokenizer and is then broken into separate string tokens. A token is a **word**.
 
-- For Latin-based languages, the words are separated by **space**.
-- For Kanji characters, the words are separated by **character**.
+- For Latin-based languages, the words are separated by **space**
```

Comment:
It's not only for Latin-based characters but for everything, Some Languages don't use any separator but Meilisearch doesn't make any exceptions.
Let's say we find a space in the middle of a Kanji string, Meilisearch will split on it as it would make for any other script.
Moreover, Latin-based languages are not the only ones using spaces, for example, the Cyrillic Script uses some of the separators used in Latin Languages.